site.yaml

```diff
@@ -34,20 +34,12 @@
     ceph_keyring: "{{ ceph_config_home }}/ceph.client.admin.keyring"
     apply_spec: false
   tasks:
-    - name: Get ceph fsid
-      command: "awk '/fsid/ {print $3}' /etc/ceph/ceph.conf"
-      register: fsid
-      run_once: true
-      changed_when: false
-
-    - name: Set fsid fact
-      set_fact:
-        fsid: "{{ fsid.stdout }}"
-
     - name: Set client fact
-      set_fact:
-        ceph_cli: "{{ cephadm_bin_home }}/cephadm shell --fsid {{ fsid }} -c {{ ceph_dot_conf }} -k {{ ceph_keyring }} -- ceph"
-      run_once: true
+      include_role:
+        name: tripleo_ceph_set_container_cli
```

Comment:
This needs to be changed from tripleo_ceph_ to tripleo_cluster_ or you get this error

```
ERROR! the role 'tripleo_ceph_set_container_cli' was not found in /home/stack/tripleo-ceph/roles:/home/s
tack/.ansible/roles:/usr/share/ansible/roles:/etc/ansible/roles:/home/stack/tripleo-ceph               

The error appears to be in '/home/stack/tripleo-ceph/site.yaml': line 39, column 15, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

      include_role:
        name: tripleo_ceph_set_container_cli
              ^ here
```

Reply:
I think I'll merge it and drop in a quick fix right after.